utils/cross-build-mlir-aie.sh

```diff
@@ -47,31 +43,21 @@ mkdir -p $INSTALL_DIR
 cd $BUILD_DIR
 
 set -o pipefail
-
+set -e
 cmake -GNinja \
-    -DBUILD_SHARED_LIBS=OFF \
-    -DCMAKE_BUILD_TYPE=Release \
-    -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR} \
-    -DCMAKE_MODULE_PATH=${CMAKEMODULES_DIR} \
-    -DCMAKE_SYSROOT=${CMAKE_SYSROOT} \
-    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE} \
+    -DCMAKE_MODULE_PATH=${CMAKEMODULES_DIR}/modulesXilinx \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKEMODULES_DIR}/toolchainFiles/toolchain_clang_crosscomp_pynq.cmake \
+    -DSysroot=${CMAKE_SYSROOT} \
+    -DArch=arm64 \
     -DLLVM_DIR=${LLVM_DIR}/build-aarch64/lib/cmake/llvm \
     -DMLIR_DIR=${LLVM_DIR}/build-aarch64/lib/cmake/mlir \
-    -DVitisSysroot=${CMAKE_SYSROOT} \
+    -DLLVM_USE_LINKER=lld \
+    -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR} \
+    -DAIE_ENABLE_BINDINGS_PYTHON=ON \
+    -DCMAKE_BUILD_TYPE=Debug \
+    -Wno-dev \
```

Comment:
Most of this should be reverted:
* The toolchain file should not be hard coded, or it should at least use the air aarch64 toolchain file.
* The build type should not be hard coded
* We don't need -DSysroot -DArch if using the air aarch64 toolchain file
* LLVM_USE_LINKER should be set by toolchain file

Reply:
> Hi Jeff, thanks for your responses. This script, which cross compiles mlir-aie was trying to be conform with mlir-aie's ARM cross compile script: https://github.com/Xilinx/mlir-aie/blob/main/utils/build-mlir-aie-cross.sh
> 
> * The toolchain file should not be hard coded, or it should at least use the air aarch64 toolchain file.
>   -- I think if mlir-aie's build process depends on an air toolchain file, then there seems to be a circular dependency where air's aarch64 toolchain -> mlir-aie repo -> mlir-air. Then when mlir-aie bumps, should air developers have to maintain aarch64 toolchain file accordingly too? I wonder if it will be possible to use aie's own build process, using its own toolchain file? Is there a fundamental difference between air's aarch64 toolchain file and aie's clang crosscomp pync toolchain file, that we in AIR must maintain?

Just don't hard code it please so that I can use my own.

Reply:
> > Hi Jeff, thanks for your responses. This script, which cross compiles mlir-aie was trying to be conform with mlir-aie's ARM cross compile script: https://github.com/Xilinx/mlir-aie/blob/main/utils/build-mlir-aie-cross.sh
> > 
> > * The toolchain file should not be hard coded, or it should at least use the air aarch64 toolchain file.
> >   -- I think if mlir-aie's build process depends on an air toolchain file, then there seems to be a circular dependency where air's aarch64 toolchain -> mlir-aie repo -> mlir-air. Then when mlir-aie bumps, should air developers have to maintain aarch64 toolchain file accordingly too? I wonder if it will be possible to use aie's own build process, using its own toolchain file? Is there a fundamental difference between air's aarch64 toolchain file and aie's clang crosscomp pync toolchain file, that we in AIR must maintain?
> 
> Just don't hard code it please so that I can use my own.

Reverted to use your own toolchain.